### ISSUE TITLE
Minor issue with documentation for TakeLast

### ISSUE THREAD

**Issue by GoogleCodeExporter** [blyzer/morelinq#69]
```
The documentation for TakeLast includes the following line:
    /// <param name="source">The sequence to pad.</param> 

That does not sound right. I suppect the documentation was based on a copy and 
paste from the Pad method, and that this line was accidently not updated.
```

Original issue reported on code.google.com by `KevinCat...@gmail.com` on 3 Aug 2011 at 7:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 28 Dec 2012 at 10:18
- Changed state: **Accepted**
- Added labels: **Component-Docs**

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 28 Dec 2012 at 10:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 28 Dec 2012 at 10:23
- Added labels: **Milestone-Release1.0**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a867a312af6e.
```

Original comment by `azizatif` on 28 Dec 2012 at 10:23
- Changed state: **Fixed**
